pkg/user/tx_client.go

```diff
@@ -46,6 +47,7 @@ type TxResponse struct {
 	Height int64
 	TxHash string
 	Code   uint32
+	Error  string
```

Comment:
I would advocate for removing error here because it should be returned in the `error` type

Reply:
done!